src/game.js

```diff
@@ -440,10 +440,12 @@ var Game = Class.create({
 
 	/*	resizeCombatFrame()
 	 *
-	 *	Resize the combat frame
+	 *	Resizes the combat frame
 	 *
 	 */
 	resizeCombatFrame: function() {
+		//TO DO to fix or to delete this unused code
```

Comment:
`// TODO: ` This way it gets highlighted in code editors.

Reply:
Thanks!